src/console/widget.ts

```diff
@@ -440,32 +440,17 @@ namespace ConsoleWidget {
    * The default implementation of an `IRenderer`.
    */
   export
-  class Renderer implements IRenderer {
+  abstract class Renderer implements IRenderer {
```

Comment:
This class is not needed since we already have the interface and it is not implementing anything concretely.